xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-default/src/main/java/org/xwiki/mentions/internal/DefaultMentionsEventExecutor.java

```diff
@@ -193,17 +193,15 @@ public void consume()
             MentionsData data = null;
             try {
                 data = DefaultMentionsEventExecutor.this.queue.take();
-                if (data != null) {
```

Comment:
Why removing this null check here?

Reply:
Because AFAIK `take()` does not return null values.